tests/inputs/service_uppercase/test_service.py

```diff
@@ -0,0 +1,7 @@
+import inspect
+from tests.output_betterproto.service_uppercase import TestStub
```

Comment:
```suggestion

from tests.output_betterproto.service_uppercase import TestStub
```
this fixes the isort failure in the CI. See also https://github.com/wouterh/python-betterproto/pull/1